templates/legal/ubuntu-pro-description/index.md

```diff
@@ -64,65 +64,71 @@ As a weekday or 24/7 support customer, you are entitled to all of the benefits o
-            17. OpenStack clouds not deployed through a [Private Cloud Build](https://ubuntu.com/pricing/consulting) or validated using the Cloud Validation Package are limited to Bug-fix Support
-            18. OpenStack support does not include support beyond Bug-fix Support during the deployment or configuration of an OpenStack cloud
-        13. <span id="uprosd-exclusions">Exclusions</span>
-            19. Full Stack Support excludes customisations which are not considered Valid Customisations
-            20. Support for workloads other than those required to run an OpenStack deployment
-            21. Support for Guest Instances other than Cloud Guests
-    9. <span id="uprosd-charms-support">Charms support</span>
-        14. Each Charm version is supported for one year from the release date
-        15. Canonical will not provide support for any Charms that have been modified from the [supported version](https://docs.openstack.org/charm-guide/latest/reference/openstack-charms.html)
-    10. <span id="uprosd-ceph-storage-support">Ceph storage support</span>
+    1. <span id="uprosd-openstack">OpenStack</span>
+        1. OpenStack software support depends on the Ubuntu release deployed on the underlying Nodes:
+            1. The [version of OpenStack provided initially in the release of a LTS version of Ubuntu](https://ubuntu.com/about/release-cycle#openstack-release-cycle) is supported for the entire lifecycle of that Ubuntu version
+            2. Releases of OpenStack after an LTS version of Ubuntu are available in the Ubuntu Cloud Archive. Each OpenStack release in the Ubuntu Cloud Archive is supported on an Ubuntu LTS version for a minimum of 18 months from the release date of the Ubuntu version that included the applicable OpenStack version
+        2. OpenStack support requires all Nodes that participate in the OpenStack Cloud to be covered under an active support agreement
+        3. OpenStack support includes access to use available Canonical provided Microsoft-certified drivers in Windows Guest instances
+        4. <span id="uprosd-full-openstack-support">Full OpenStack support</span>
+            1. Requirements:
+                1. OpenStack Charms -based deployments:
+                   1. In addition to the requirements set out above, hardware must meet the minimum criteria as specified by Canonical as part of the [Private Cloud Build](https://ubuntu.com/openstack/consulting) or [Cloud Validation](https://ubuntu.com/openstack/consulting) consulting engagements
+                   2. The OpenStack Cloud was deployed via a [Private Cloud Build](https://ubuntu.com/openstack/consulting) or was validated by Canonical through [Cloud Validation](https://ubuntu.com/openstack/consulting)
+                2. MicroStack-based deployments:
+                   1. In addition to the requirements set out above, hardware must meet the minimum criteria as specified in [MicroStack documentation](https://microstack.run/docs)
```

Comment:
```suggestion
                   1. In addition to the requirements set out above, hardware must meet the minimum criteria as specified in [MicroStack documentation](https://microstack.run/docs/enterprise-reqs)
```

Reply:
What I can see on the doc is just `/docs` not `/docs/enterprise-reqs`